cmd/flags_rpc.go

```diff
@@ -18,12 +18,12 @@ func RPCFlags() *flag.FlagSet {
 
 	flags.String(
 		addrFlag,
-		"",
+		"0.0.0.0",
 		"Set a custom RPC listen address (default: localhost)",
 	)
 	flags.String(
 		portFlag,
-		"",
+		"26658",
```

Comment:
Value is hardcoded in 2 places - here and in https://github.com/celestiaorg/celestia-node/blob/62697fd56ec2b732b3a43afaf9bcad61d139263f/node/rpc/config.go#L12

Reply:
`DefaultConfig()` can be reused here, as a source of "defaults".

Reply:
https://github.com/celestiaorg/celestia-node/pull/696/commits/05163a55c4196b1db89e6f5ea39298691e179fd5